package.json

```diff
@@ -32,7 +32,7 @@
   "devDependencies": {
     "http-server": "^0.12.3",
     "parcel-bundler": "^1.12.4",
-    "purescript": "^0.13.6",
+    "purescript": "^0.13.8",
```

Comment:
I'd like to make this update, but Zephyr doesn't work with PureScript 0.13.8 because of the [change to store externs in a binary format](https://github.com/purescript/purescript/pull/3841). I'd like to wait until Zephyr can be updated before switching to 0.13.8

Reply:
Thanks for linking to the PR. I was trying to track down whether `cbor` was new in `v0.13.8`. I figured that was the cause of the issue I had with zephyr, but hadn't come across that PR yet.

Reply:
Well, Zephyr was just [updated an hour ago](https://github.com/coot/zephyr/pull/34) to accommodate the new format! There hasn't been a release yet but there should be one soon.

Reply:
I have purescript 0.13.8 installed globally. I can't seem to install 0.13.6 as a dev dependency in halogen-realworld - it always resolves to 0.13.8. That seems surprising to me. I did change the version back to `"^0.13.6"`, then ran `npm run clean` and manually deleted `package-lock.json` and other build/installation artifacts I could find before running `npm i` again.

Not requesting support here - I just wanted to mention it in case it's not solely an issue with my local setup.

Reply:
Not able to do `npm i --save-dev purescript@0.13.6`?

Reply:
Good call. I hadn't tried that. I'd just tried manually updating the version in `package.json` and running a full `npm i`, which I think should work, but...

`npm i --save-dev purescript@0.13.6` did the trick, though. Apologies for the bother.